Modules/socketmodule.c

```diff
@@ -305,6 +305,69 @@ if_indextoname(index) -- return the corresponding interface name\n\
 /* Provides the IsWindows7SP1OrGreater() function */
 #include <VersionHelpers.h>
 
+/* This block removes some flags on older version Windows during run-time.
+   https://msdn.microsoft.com/en-us/library/windows/desktop/ms738596.aspx */
+#if 1
+typedef struct {
+    DWORD build_number;  /* available starting with this Win10 BuildNumber */
+    const char flag_name[20];
+} FlagRuntimeInfo;
+
+/* IMPORTANT: make sure the list ordered by descending build_number */
+static FlagRuntimeInfo flags[] = {
```

Comment:
This is a big module - using a name more like `tcp_flag_windows_version` will avoid conflicts with other parts.